### PR TITLE
(dbt) Migration enriched venue data

### PR DESCRIPTION
**Pull request by carolinemestre-passculture**
# DE PR

## Describe your changes

Le but de cette PR est de migrer le modèle `enriched_venue_data` , le nouveau nom du modèle sera `mrt_global__venue` et les deux tables devront être iso.

Nous migrons également tous les modèles nécessaires à la construction de ce modèle.

Tag a reviewer if necessacy @LucileRainteau @valoumiaou @MatthieuRipollPass 

## Jira ticket number and/or notion link

JIRA-ticket_number

### Type of change
- [ ] Fix (non-breaking change which corrects expected behavior)
- [ ] New fields (non-breaking change)
- [x] New table (non-breaking change)
- [ ] Concept change (potentially breaking change which modifies fields according to new or evolving business concepts) 
- [ ] Table deletion (potentially breaking change which adds functionality/ table)
      
### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] My code passes CI/CD tests
- [ ] I updated README.md
- [ ] I have updated the dag
- [ ] If my changes concern incremental table, I have altered their schema to accomodate with field's creation/deletion
- [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
- [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
- [ ] I will create a review on slack and ensure to specify the duration of the review task: short (<10min), medium (<30min), long (>30min)

### Added tests?
- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] ⏰ no, but I created a ticket

### CODE REVIEW

File: orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__booking.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    booking_used_date,
+    booking_cancellation_date,
+    booking_cancellation_reason,
+    deposit_id,
+    offerer_id,
+    venue_id,
+    price_category_label,
+    booking_reimbursement_date,
+    coalesce(b.booking_amount, 0) * coalesce(b.booking_quantity, 0) AS booking_intermediary_amount,
+    rank() OVER (PARTITION BY b.user_id ORDER BY b.booking_creation_date) AS booking_rank,
```
Comment (MatthieuRipollPassCulture) [inline orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__booking.sql:21]: ROW_NUMBER() par précaution ?

File: orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__collective_booking.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    cb.collective_booking_cancellation_limit_date,
+    cb.collective_booking_cancellation_reason,
+    cb.collective_booking_status,
+    cb.collective_booking_reimbursement_date,
+    cb.educational_institution_id,
+    cb.educational_year_id,
+    cb.collective_booking_confirmation_date,
+    cb.collective_booking_confirmation_limit_date,
+    cb.educational_redactor_id,
+    rank() OVER (
```
Comment (MatthieuRipollPassCulture) [inline orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__collective_booking.sql:18]: ROW_NUMBER() par précaution ?

File: orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__booking.sql (hunk 1 above)
Comment (carolinemestre-passculture) [reply orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__booking.sql:21]: Je n'ai pas touché à la logique des modèles pour faciliter la partie sur les data quality checks. Si on veut faire ce changement, il faudrait le faudrait aussi dans `orchestration/dags/data_gcp_dbt/models/clean/applicative/booking.sql`.

File: orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__collective_booking.sql (hunk 2 above)
Comment (carolinemestre-passculture) [reply orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__collective_booking.sql:18]: Idem que pour `int_applicative__booking`.

File: orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__booking.sql (hunk 1 above)
Comment (valoumiaou) [reply orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__booking.sql:21]: finalement on verra ca plus tard pour que les données soient iso et assurer les check de migration.
j'ai fais un ticket backlog DA https://passculture.atlassian.net/browse/PC-28627

File: orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__offer.sql [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        FROM {{ ref('int_applicative__stock') }}
+        GROUP BY offer_id
+)
+
+SELECT
+    o.offer_id,
+    o.offer_id_at_providers,
+    o.offer_modified_at_last_provider_date,
+    DATE(o.offer_creation_date) AS offer_creation_date,
+    o.offer_creation_date AS offer_created_at,
```
Comment (carolinemestre-passculture) [inline orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__offer.sql:44]: @valoumiaou @LucileRainteau vous en pensez quoi de cette naming convention pour les date vs les timestamps?

Comment (LucileRainteau) [reply orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__offer.sql:30]: Ca me vaaa !

**Review by LucileRainteau — DISMISSED** [review]

**Review by LucileRainteau — APPROVED** [review]

**Review by LucileRainteau — APPROVED** [review]

**Pull request merged**